systemtest/src/main/java/io/strimzi/systemtest/resources/ResourceManager.java

```diff
@@ -426,6 +430,30 @@ public static String getImageValueFromCO(String name) {
         return resource;
     }
 
+    public static <T extends HasMetadata> long getResourceOperationTimeout(String kind, ResourceOperation operation) {
+        return getResourceOperationTimeout(kind, operation, 0);
+    }
+
+    public static <T extends HasMetadata> long getResourceOperationTimeout(String kind, ResourceOperation operation, int podsToRoll) {
```

Comment:
Where do you use `T`?

Reply:
Oops, didn't remove it after last change, thanks!